pkg/client/client.go

```diff
@@ -63,16 +66,18 @@ func newWorker(p *properties.Properties, threadID int, threadCount int, workload
 			totalOpCount = p.GetInt64(prop.RecordCount, 0)
 		}
 	}
-
-	if totalOpCount < int64(threadCount) {
-		fmt.Printf("totalOpCount(%s/%s/%s): %d should be bigger than threadCount: %d",
-			prop.OperationCount,
-			prop.InsertCount,
-			prop.RecordCount,
-			totalOpCount,
-			threadCount)
-
-		os.Exit(-1)
+	//Keep data running
+	if !p.GetBool(prop.NormalDataInTime, prop.NormalDataInTimeDefault) {
```

Comment:
Why should keep it running forever? IMO, if you must let it running forever, you can change totalOpCount, it's better than adding a strange if predication statement here.